yt/frontends/enzo_e/data_structures.py

```diff
@@ -475,7 +475,7 @@ def _set_code_unit_attributes(self):
             setdefaultattr(self, "velocity_unit", self.quan(k["uvel"], "cm/s"))
         else:
             p = self.parameters
-            for d, u in zip(("length", "time"), ("cm", "s")):
+            for d, u in [("length", "cm"), ("time", "s")]:
```

Comment:
I can't help but feel like I probably initiated this idiom that you're undoing in several places ... and I think undoing it is the right thing!